beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -122,14 +122,16 @@ lazy_static! {
     /*
      * Block Statistics
      */
-    pub static ref OPERATIONS_PER_BLOCK_ATTESTATION: Result<Histogram> = try_create_histogram(
+    pub static ref OPERATIONS_PER_BLOCK_ATTESTATION: Result<Histogram> = try_create_histogram_with_buckets(
         "beacon_operations_per_block_attestation_total",
-        "Number of attestations in a block"
+        "Number of attestations in a block",
+        decimal_buckets(0,1)
```

Comment:
`decimal buckets(0, 2)` might be better so that we get `[1, 2, 5, 10, 20, 50, 100, 200, 500]`

or maybe something that allows us to distinguish full blocks (128) from others, maybe exponential buckets with a 1 offset like `[0, 1, 3, 15, 31, 63, 127, 255]`?

Reply:
nts: maybe some values between 127 and 255?